### PR TITLE
error handling for Panic in mesheryctl env create 

### LINKED ISSUE #11314: [mesheryctl] `environment create ` command panic error
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior

When running `mesheryctl exp environment create --orgId [orgID] -n [name] --description [description]` returns panic.
### Desired Behavior

- [ ] Fix panic error.
- [ ] Update the `orgID` field from `orgId`

### Screenshots/Logs

```
./mesheryctl exp environment create --orgId b42adb3e-26dd-4668-aea7-bf040d872023 --name shlok --description hello 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x3b7258e]

goroutine 1 [running]:
github.com/layer5io/meshery/mesheryctl/internal/cli/root/environments.glob..func3(0xc00050d700?, {0x459a580?, 0x4?, 0x459a584?})
        /home/jougan/OpenSource/meshery/mesheryctl/internal/cli/root/environments/create.go:112 +0x20e
github.com/spf13/cobra.(*Command).execute(0x66419e0, {0xc000981440, 0x6, 0x6})
        /home/jougan/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0xabc
github.com/spf13/cobra.(*Command).ExecuteC(0x66333e0)
        /home/jougan/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        /home/jougan/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/layer5io/meshery/mesheryctl/internal/cli/root.Execute()
        /home/jougan/OpenSource/meshery/mesheryctl/internal/cli/root/root.go:85 +0x45
main.main()
        /home/jougan/OpenSource/meshery/mesheryctl/cmd/mesheryctl/main.go:26 +0x13
```

### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by pranjalg1331**
**Notes for Reviewers**

This PR fixes #11314 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/environments/create.go [hunk 1]
```diff
@@ -84,7 +84,10 @@ https://docs.layer5.io/cloud/spaces/environments/
 			return err
 		}
 
-		resp, _ := utils.MakeRequest(req)
+		resp, err := utils.MakeRequest(req)
+		if err != nil {
+			return err
```
Comment (lekaf974) [inline mesheryctl/internal/cli/root/environments/create.go:89]: Can you use a meshkit errors instead ref: https://docs.meshery.io/project/contributing/contributing-error

**Comment by vishalvivekm** [comment]
@pranjalg1331 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

**Review by lekaf974 — APPROVED** [review]

**Comment by pranjalg1331** [comment]
@lekaf974 Can i close this pr as the code is already merged?

**Comment by pranjalg1331** [comment]
@lekaf974 can we merge this pr with the master?

**Comment by lekaf974** [comment]
> @lekaf974 can we merge this pr with the master?

Did this PR been presented in Dev meeting ?

**Comment by pranjalg1331** [comment]
@lekaf974 yes the pr has been presented in the dev meeting. I have already commited the suggested changes.

**Pull request merged**